src/azure-cli/azure/cli/command_modules/appconfig/_utils.py

```diff
@@ -254,3 +261,73 @@ def parse_tags_to_dict(tags):
                 tags_dict[tag_key] = tag_value
         return tags_dict
     return tags
+
+
+def strip_json_comments(json_string):
```

Comment:
If we do go with this, see by below comment, is there a reason why we aren't using the same code as the provider? There shouldn't be any difference in what they are doing.

Reply:
See my comment below, where I give another idea.